app/views/candidates/schools/_phase2.html.erb

```diff
@@ -206,7 +206,7 @@
         <%= safe_format @presenter.teacher_training_info %>
 
         <p>
-          <%= link_to 'More information', @presenter.teacher_training_url %>
+          <%= link_to 'Find out more about our teacher training', @presenter.teacher_training_url %>
```

Comment:
We also need to reflect this content change in the profile preview on MSE.

![Screenshot 2021-12-06 at 16 24 56](https://user-images.githubusercontent.com/85290597/144883564-1ce314fa-afc9-41d9-af40-f0e2aa5ef2a0.png)

Reply:
Just pushed